module/zfs/dmu_object.c

```diff
@@ -31,6 +31,8 @@
 #include <sys/zap.h>
 #include <sys/zfeature.h>
 
+int zfs_metadnode_backfill = 0;
```

Comment:
You mean to make it a zpool property?  I think that would be a bad idea, since this is essentially a workaround for a performance bug.